ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -30,9 +30,17 @@ sub valid_sshd_file
 {
     my ($self, $file, $cfg) = @_;
 
+    my $sshd_bin = $cfg->{sshd_path} || DEFAULT_SSHD_PATH;
+    my $validate_optional = $cfg->{validate_optional};
+    
+    if (defined($validate_optional) && $validate_optional && ! -x $sshd_bin) {
+        $self->info("$sshd_bin doesn't exist, skipping sshd config test.");
```

Comment:
maybe add `with validate_optional=1` (the message best expresses why this is skipped, and it's not only because sshd is missing)